tests/common/test_schema.py

```diff
@@ -67,3 +79,37 @@ def test_drop_descriptions():
     }
     drop_schema_descriptions(node)
     assert node == expected
+
+
+################################################################################
+# Test call transaction schema
+
+
+def test_validate_transaction_create(create_tx):
```

Comment:
The test passes if no validation error is thrown. Any exception will fail a test.